app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -2,19 +2,27 @@
   # We have to call deep_symbolize_keys because we're often dealing with a
   # parsed JSON document which will have string keys by default, but our
   # components use symbol keys and we want consistency.
-  links_hash = content_item.to_h.deep_symbolize_keys[:links] || {}
+  content_item_hash = content_item.to_h.deep_symbolize_keys
+  links_hash = content_item_hash[:links] || {}
```

Comment:
We're keeping the language constructs used in these templates as generic as possible so that they could be ported to another templating language - @dsingleton can explain the rationale for that.